### PR TITLE
Allow passing a properties file to the quickstart examples

### PR DESCRIPTION
**Pull request by mimaison**
Added propertiesFile argument to specify connection settings

### CODE REVIEW

**Comment by mimaison** [comment]
cc @apurvam

File: ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java [hunk 1]
```diff
@@ -102,7 +111,8 @@ private static void usage() {
         + "topic=<kafka topic name> "
         + "key=<name of key column> "
         + "[iterations=<number of rows> (defaults to 1,000,000)] "
-        + "[maxInterval=<Max time in ms between rows> (defaults to 500)]"
+        + "[maxInterval=<Max time in ms between rows> (defaults to 500)] "
+        + "[propertiesFile=<file specifying Kafka client properties]"
```
Comment (apurvam) [inline ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java]: replace `]` with `>`

Comment (mimaison) [reply ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java]: Good catch, actually it needs to be `>]`

**Comment by apurvam** [comment]
cc @hjafarpour @rodesai.. we need one more LGTM before merging.

**Comment by dguy** [comment]
Thanks @mimaison, the build is failing with checkstyle errors:
```
[ERROR] /home/****/workspace/confluentinc-pr_ksql_PR-657-STDDV3EDCIRYTDPINBVPOZLDSDGBA7A5PROO5OTQHOEAKBCXG5GA/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java:89: 'if' have incorrect indentation level 8, expected level should be 6. [Indentation]
[ERROR] /home/****/workspace/confluentinc-pr_ksql_PR-657-STDDV3EDCIRYTDPINBVPOZLDSDGBA7A5PROO5OTQHOEAKBCXG5GA/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java:90: 'if' child have incorrect indentation level 12, expected level should be 8. [Indentation]
[ERROR] /home/****/workspace/confluentinc-pr_ksql_PR-657-STDDV3EDCIRYTDPINBVPOZLDSDGBA7A5PROO5OTQHOEAKBCXG5GA/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java:91: 'if rcurly' have incorrect indentation level 8, expected level should be 6. [Indentation]
[ERROR] /home/****/workspace/confluentinc-pr_ksql_PR-657-STDDV3EDCIRYTDPINBVPOZLDSDGBA7A5PROO5OTQHOEAKBCXG5GA/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java:93: 'catch' child have incorrect indentation level 8, expected level should be 6. [Indentation]
[ERROR] /home/****/workspace/confluentinc-pr_ksql_PR-657-STDDV3EDCIRYTDPINBVPOZLDSDGBA7A5PROO5OTQHOEAKBCXG5GA/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java:94: 'catch' child have incorrect indentation level 8, expected level should be 6. [Indentation]
```

**Pull request merged**